<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Billing Handler -- Semi Automatic</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>导出文件名称</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="OutFilename">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Sheet名称(不填则默认为当前active)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="SheetName"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>手机号码 Column</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="PhoneColumn"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>花费 Column</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="CostColumn"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>小计 Column(无需小计则不填)</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLineEdit" name="CountColumn"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>导入文件名称</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLineEdit" name="InFilename">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>提示信息（成功/输入有误）</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QTextBrowser" name="resultBrowser"/>
   </item>
   <item row="14" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>启动</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
